base/metrics/persistent_memory_allocator.cc

```diff
@@ -320,7 +320,7 @@ PersistentMemoryAllocator::PersistentMemoryAllocator(Memory memory,
       mem_type_(memory.type),
       mem_size_(checked_cast<uint32_t>(size)),
       mem_page_(checked_cast<uint32_t>((page_size ? page_size : size))),
-#if BUILDFLAG(IS_NACL)
+#if BUILDFLAG(IS_NACL) || defined(STARBOARD)
```

Comment:
Tiny nit, but perhaps using the other codepath here is cleaner, in ensuring SysInfo {} is available for Starboard ? We could perhaps just stub it or hardcode constants and add NOTIMPLEMENTED to functions that we are sure we don't want to implement.

Reply:
Starboard actually implements this function and returns a 0. Probably it should just return 4096U.